front/cypress/e2e/project_page_native_survey.cy.ts

```diff
@@ -175,3 +175,38 @@ describe('Archived continuous project with survey', () => {
     cy.apiRemoveProject(projectId);
   });
 });
+
+describe('Modal shown after survey submission', () => {
```

Comment:
Please double check that we have a test for submission of an idea. It not, please add it, because we have to be sure that the behaviour after submitting an idea has not changed by this PR.

Reply:
Ok I'll double check and add one if it's missing.

Reply:
We do indeed have an e2e test for this - called _idea_new_page_ which verifies the content of the newly created idea page (post submission). It passes on this branch :+1: